src/storeTvlInterval/storeNewTokensValueLocked.ts

```diff
@@ -0,0 +1,14 @@
+import dynamodb from "../utils/dynamodb";
+import { Protocol } from "../protocols/data";
+import { getTimestampAtStartOfDay } from "../utils/date";
+import { TokensValueLocked } from "../types";
+
+export default async (protocol: Protocol,
+  unixTimestamp: number,
+  tokensVL: TokensValueLocked) => {
+  await dynamodb.put({
+    PK: `dailyTokensVL#${protocol.id}`,
```

Comment:
typo?

Reply:
what do you mean?

Reply:
I was just wondering if `dailyTokensVL` should be `dailyTokensTVL`?